docs/output.md

```diff
@@ -1,9 +1,5 @@
 # nf-core/nanoseq: Output
 
-## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/nanoseq/output](https://nf-co.re/nanoseq/output)
```

Comment:
Is this removal from the template? :thinking:

Reply:
I have added this back in